src/applications/virtual-agent/tests/Chatbox.unit.spec.js

```diff
@@ -69,6 +69,17 @@ describe.skip('App', () => {
     );
   }
 
+  async function wait(timeout) {
+    return new Promise(resolve => {
+      setTimeout(resolve, timeout);
```

Comment:
Should we be using timeouts in the tests now?

Reply:
It's a different usage in this case. The actual elapse of time is all timestamp based, using normal JS date functions.  
In this case it's getting called for a tenth of a second (not an hour). It seems the breaking changes were a result of using useFakeTimers, and the tick() call was not working for some reason.

Reply:
It's a different usage in this case. The actual elapse of time is all timestamp based, using normal JS date functions.
In this case it's getting called for a tenth of a second (not an hour). It seems the breaking changes were a result of using useFakeTimers, and the tick() call was not working for some reason.